src/controllers/organizationController.ts

```diff
@@ -314,16 +280,61 @@ export default class OrganizationController {
                 )
             )
           )
-
-          .map(requestIdCollection => ({
-            requestIds: requestIdCollection.ids,
-            user: taskResults.user
-          }))
+          .map(actionPayload => ({ actionPayload, user: taskResults.user }))
       )
-      .chain<Pick<ITaskResults, "requestModels">>(taskResults =>
-        array
-          .traverse(taskEither)([...taskResults.requestIds], requestId =>
-            this.getValidRequest(requestId, taskResults.user.email)
+      .chain(taskResults => {
+        if (
+          taskResults.actionPayload.type ===
+          RequestActionEnum.SEND_REGISTRATION_REQUEST_EMAIL_TO_ORG
+        ) {
+          return this.sendEmailWithDocumentsToOrganizationPec(
+            taskResults.user,
+            taskResults.actionPayload.ids
+          );
+        }
+        const unhandledActionMessage = "Unhandled action.";
+        log.error(
+          `organizationController#handleAction | ${unhandledActionMessage}`
+        );
+        return fromLeft<IResponseErrorInternal, IResponseNoContent>(
```

Comment:
this is a 404 not found or 400 bad request more than internal error

apart from this, lgtm